Main.rb

```diff
@@ -0,0 +1,19 @@
+dirPath = File.dirname(File.dirname(File.expand_path(__FILE__)))
+load dirPath + "/CoffeeShopApp/Objects/CoffeeShop.rb"
```

Comment:
When I try to run the code via

```
ruby Main.rb
```

I get the following error

Traceback (most recent call last):
        1: from Main.rb:2:in `<main>'
Main.rb:2:in `load': cannot load such file -- /<path>/CoffeeShopApp/Objects/CoffeeShop.rb (LoadError)
```

Please take a look at [require](https://ruby-doc.org/core-2.6.5/Kernel.html#method-i-require) and [require_relative](https://ruby-doc.org/core-2.6.5/Kernel.html#method-i-require_relative) kernel methods.